website/pages/docs/commands/operator/snapshot-agent.mdx

```diff
@@ -0,0 +1,149 @@
+---
+layout: docs
+page_title: 'Commands: operator snapshot agent'
+sidebar_title: snapshot agent
+description: |
+  Periodically saves snapshots of Nomad server state
+---
+
+# Command: operator snapshot agent
+
+<EnterpriseAlert />
+
+The snapshot agent takes snapshots of the state of the nomad servers and
+saves them locally, or pushes them to an optional remote storage service.
+
+The agent can be run as a long-running daemon process or in a one-shot mode
+from a batch job. As a long-running daemon, the agent will perform a leader
+election so multiple processes can be run in a highly available fashion with
+automatic failover. In daemon mode, the agent will also register itself with
+Nomad as a service, along with health checks that show the agent is alive
+and able to take snapshots.
+
+If ACLs are enabled, a management token must be supplied in order to perform
+snapshot operations.
+
+The Config file has the following format (shown populated with default values):
+
+```hcl
+nomad {
+  http_addr       = "127.0.0.1:8500"
+  token           = ""
+  region          = ""
+  ca_file         = ""
+  ca_path         = ""
+  cert_file       = ""
+  key_file        = ""
+  tls_server_name = ""
+}
+
+
+snapshot {
+  interval         = "1h"
+  retain           = 30
+  stale            = false
+  service          = "nomad-snapshot"
+  deregister_after = "72h"
+  lock_key         = "nomad-snapshot/lock"
+  max_failures     = 3
+  name_prefix      = "nomad"
+}
+
+log {
+  level           = "INFO"
+  enable_syslog   = false
+  syslog_facility = "LOCAL0"
+}
+
+consul {
```

Comment:
```suggestion
nomad {
```

Reply:
This one is actually meant to be consul - nomad's block is above - agent can be configured to use consul for service discovery and locking.